pl_bolts/models/detection/faster_rcnn/faster_rcnn_module.py

```diff
@@ -90,13 +90,22 @@ def __init__(
             self.model.roi_heads.box_predictor = FastRCNNPredictor(in_features, self.num_classes)
 
         else:
-            backbone_model = create_fasterrcnn_backbone(
-                self.backbone,
-                fpn,
-                pretrained_backbone,
-                trainable_backbone_layers,
-                **kwargs,
-            )
+            if isinstance(self.backbone, torch.nn.Module):
+                backbone_model = self.backbone
+                if pretrained_backbone:
```

Comment:
shall we add some check on the required dimensionality, I believe that you cannot add any backbone, right?

Reply:
@abhayraw1 ^^

Reply:
To unblock this PR, I think this is fine for now :)